lib/mongodb.js

```diff
@@ -23,13 +22,14 @@ const ObjectIdValueRegex = /^[0-9a-fA-F]{24}$/;
 const ObjectIdTypeRegex = /objectid/i;
 
 exports.ObjectID = ObjectID;
+exports.ObjectId = ObjectID;
```

Comment:
Yes, bson renamed this to match class names. See https://github.com/mongodb/js-bson/blob/main/src/bson.ts#L92-L95. I think that is also a good idea to start using (or at least export) new class name since this changed at new bson versions.